docs/src/common_mlj_workflows.md

```diff
@@ -14,9 +14,9 @@ channing = (Sex = rand(["Male","Female"], 462),
 coerce!(channing, :Sex => Multiclass)
 ```
 
-```julia
-import RDatasets
-channing = RDatasets.dataset("boot", "channing")
+```julia-repl
+julia> import RDatasets
+julia> channing = RDatasets.dataset("boot", "channing")
```

Comment:
```suggestion
import RDatasets
channing = RDatasets.dataset("boot", "channing")
```

I think for consistency, we just skip all the `julia> ` prompts in common_mlj_workflows.jl`, no?

Reply:
Sounds good. It should be noted that when you copy paste into the REPL, as far as I've noticed (Julia 1.10) it ignores the prompt if it's from the clipboard. But for example workflows (without showing output) consistent lack of prompt also works.